### ISSUE TITLE
Dosn't work after RestSharp update

### ISSUE THREAD

**Issue by Solnake** [gocardless/gocardless-legacy-dotnet#34]
When update RestSharp library to the latest version (105.0.1) - Sdk doesn't work.
It because they changed type of property BaseUrl in class RestClient from string to System.Uri

**Comment by jonfree** [comment]
Have you fixed the issue though or not?

**Comment by Solnake** [comment]
Yep, but this fix will require the latest version RestSharp library.
In older version the code looks like 
new RestClient
                             {
                                 BaseUrl = ApiClient.ApiUrl,
                                 UserAgent = GoCardless.UserAgent
                             };

In the latest version should be 
new RestClient
                             {
                                 BaseUrl = new Uri(ApiClient.ApiUrl),
                                 UserAgent = GoCardless.UserAgent
                             };

So, after update NuGet package should require RestSharp >= 105.0.1
